### ISSUE TITLE
Delete key?

### ISSUE THREAD

**Issue by andysay** [microcipcip/cookie-universal#1]
this.$cookies.delete('key');
? not work

**Comment by microcipcip** [comment]
Thanks for trying my script, I haven't written the documentation yet! You need to use: 
`this.$cookies.remove('key')`

**Comment by andysay** [comment]
this.$cookies.remove('key')
Not clear cookies ..

**Comment by microcipcip** [comment]
I need more contest, where are you using the module, the server or the client? What are you trying to accomplish?

**Comment by andysay** [comment]
<button @click=""clear> </button>
	methods: {


		clear(){

              this.$cookies.remove('key')



		},
not worked

**Comment by microcipcip** [comment]
Thanks, I'll test it now and see if I can reproduce the issue

**Comment by microcipcip** [comment]
I cannot reproduce your issue so far :(! I have tested it on Firefox and Chrome on windows and it is working fine, I have also created a test on codepen, can you check if it works for you? https://codepen.io/microcipcip/pen/xYgayz

P.S. is it possible that you are browsing in incognito mode?

**Comment by andysay** [comment]
I'm test in normal browsing mode , i'm use ssr nuxt js-> if i add cookie          
     this.$cookies.set('key', JSON.stringify(oldItems), {}) -> response okey data inside 

but on another page i have button with method clear 

              this.$cookies.remove('key')) - < data not remove 

nothing , data not want remove .

**Comment by andysay** [comment]
My solustion: 

mounted : {
var delete_cookie = function(name) {
    document.cookie = name + '=;expires=Thu, 01 Jan 1970 00:00:01 GMT;';
};
delete_cookie('key');
}

**Comment by microcipcip** [comment]
I think the problem is that you are setting the path value as an empty object instead of '/', let me try again and see if this is indeed the issue.

**Comment by andysay** [comment]
I'm try

**Comment by microcipcip** [comment]
Ok, I think I am pretty sure that is the issue, you either don't provide the options object and then the path will be set to `/` or you specify it, so:
```
// wrong
this.$cookies.set('key', JSON.stringify(oldItems), {})
// correct
this.$cookies.set('key', JSON.stringify(oldItems), { path: '/' })
// also correct
this.$cookies.set('key', JSON.stringify(oldItems)) // default to path: '/'
```
Because if you do not set the path, it will add the cookie to the current path you're in and therefore it won't be able to delete it.

**Comment by microcipcip** [comment]
BTW I published a new version of `cookie-universal-nuxt` you may not need JSON.stringify(oldItems) anymore, try to update and use:
```
this.$cookies.set('key', oldItems)
```

**Comment by andysay** [comment]
@microcipcip good ! now works fine :)

**Comment by andysay** [comment]
after update, broken my function

I have next function with my oldItems



              var  oldItems = await this.$cookies.get('key') || []; // < data true

              var q = this.products[0].id; // -< id data true


              for(var i = 0; i < oldItems.length; i++) {
        
                 var item = oldItems.find(item => item.id === q);

                 alert(item) < nothing broken

                if (item == oldItems[i]) {
                  oldItems[i] = addCart;
                  isReplaced = true;
                  break;
                  }  



              }

**Comment by andysay** [comment]
@microcipcip 

thank you !
